gnocchi/carbonara.py

```diff
@@ -85,6 +84,42 @@ def round_timestamp(ts, freq):
         (ts - UNIX_UNIVERSAL_START64) / freq) * freq
 
 
+TIMESERIES_ARRAY_DTYPE = [('timestamps', 'datetime64[ns]'),
+                          ('values', 'float64')]
+
+
+def make_timeseries(timestamps, values):
```

Comment:
Shouldn't base timeseries creation and manipulation hold into a class for clarity ?

I known you prefer functional code :)

But that may help to read the code.